inc/altis-config/class-qm-output-html-altis-config.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Altis Config data output for HTML pages in the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools;
+
+use function Altis\get_config;
+use QM_Collector;
+use QM_Output_Html;
+
+class QM_Output_Html_Altis_Config extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+		add_filter( 'qm/output/panel_menus', [ $this, 'panel_menu' ], 50 );
+	}
+
+	public function output() {
+		$config = get_config()['modules'] ?? [];
+
+		if ( ! $config ) {
+			echo esc_html_x( 'Something went wrong with displaying the Altis config settings.', 'Displaying Altis config info in the Query Monitor plugin', 'altis' );
+			return;
+		}
+		?>
+
+		<?php $this->before_tabular_output(); ?>
+
+		<caption>
+			<h2><?php echo esc_html( $this->name() ); ?></h2>
+		</caption>
+		<thead>
+		<tr>
+			<th><?php echo esc_html__( 'Module', 'altis' ) ?></th>
+			<th><?php echo esc_html__( 'Settings', 'altis' ) ?></th>
```

Comment:
Might be good to show the environment specific settings too in another column so we can see how they've affected the defaults. Add a column called "Environment Settings" and get the output from `get_config()['environments'][ get_environment_type() ]['modules']`